lab_1_classify_by_unigrams/main.py

```diff
@@ -64,12 +85,21 @@ def calculate_mse(predicted: list, actual: list) -> float | None:
 
     In case of corrupt input arguments, None is returned
     """
+    if len(predicted) != len(actual):
+        return None
+    if not (all(isinstance(p, (float, int)) for p in predicted) and all(isinstance(a, (float, int))
+                                                                        for a in actual)):
+        return None
+    mse = 0
+    quantity = len(actual)
+    for i in range(quantity):
```

Comment:
enumerate

Reply:
how can i use `enumerate` for this case? i need only index, not value, or vice versa